integration/integrate-with-pagerduty.adoc

```diff
@@ -0,0 +1,22 @@
+[id="integrate-with-pagerduty"]
+= Integrating with PagerDuty
+include::modules/common-attributes.adoc[]
+:context: integrate-with-pagerduty
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://www.pagerduty.com/[PagerDuty], you can forward alerts from {product-title} to PagerDuty.
+
+High-level integration flow:
+
+. Add a new API service in PagerDuty and get the integration key.
+. Use the integration key to setup notifications in {product-title}.
+. Identify policies for which you want to send notifications, and update the notification settings for those policies.
```

Comment:
"Identify policies that you want to send notifications for..." is easier for me to read.